addons/hr_recruitment/views/hr_job_views.xml

```diff
@@ -32,12 +42,12 @@
                                     <div class="o_primary col-12">
                                         <span><t t-esc="record.name.value"/></span>
                                     </div>
-                                    <div class="o_kanban_record_subtitle col-12 text-muted">
+                                    <div class="o_kanban_record_subtitle text-muted">
                                         <field name="user_id" />
                                     </div>
                                     <field name="is_favorite" widget="boolean_favorite" nolabel="1"/>
-                                    <div t-if="record.alias_name.value and record.alias_domain.value and record.state.raw_value == 'recruit'" class="o_secondary o_job_alias">
```

Comment:
Why did you remove the t-if here?

Reply:
I've removed it for testing cause I needed to verify that the email and responsible name are aligned and have same size and typo. But I did put it back in a later commit normally